mkosi/__init__.py

```diff
@@ -3152,8 +3152,12 @@ def invoke_repart(state: MkosiState, skip: Sequence[str] = [], split: bool = Fal
 
     cmdline += ["--definitions", definitions]
 
+    env = dict(TMPDIR=str(state.workspace))
+    for fs, options in state.installer.filesystem_options(state).items():
+        env[f"SYSTEMD_REPART_MKFS_OPTIONS_{fs.capitalize()}"] = " ".join(options)
```

Comment:
Do you mean?
```suggestion
        env[f"SYSTEMD_REPART_MKFS_OPTIONS_{fs.upper()}"] = " ".join(options)
```
I haven't looked at the PR for repart too closely, but related variables are `SYSTEMD_HOME_MKFS_OPTIONS_XFS` and this will only produce e.g. `SYSTEMD_REPART_MKFS_OPTIONS_Xfs`

Reply:
Ah oops